packages/xo-server/src/api/xosan.js

```diff
@@ -346,15 +346,10 @@ async function remoteSsh (glusterEndpoint, cmd, ignoreError = false) {
         }
       }
     }
-    debug(
-      result.command.join(' '),
-      '\n  =>exit:',
-      result.exit,
-      '\n  =>err :',
-      result.stderr,
-      '\n  =>out (1000 chars) :',
-      result.stdout.substring(0, 1000)
-    )
+    log.debug(result.command.join(' '))
+    log.debug(`=>exit: ${result.exit}`)
+    log.debug(`=>err: ${result.stderr}`)
+    log.debug(`=>out (1000 chars) : ${result.stdout.substring(0, 1000)}`)
```

Comment:
A log is a self-contained entity, do not use it to “print lines”.

Feel free to use the `data` object to add extra info.